usr/share/rear/restore/default/500_selinux_autorelabel.sh

```diff
@@ -12,3 +12,22 @@ is_true "$BACKUP_SELINUX_DISABLE" || return 0
 # Force SELinux relabeling of all files after reboot of the recovered system:
 touch $TARGET_FS_ROOT/.autorelabel
 LogPrint "Created SELinux $TARGET_FS_ROOT/.autorelabel file : after reboot SELinux will relabel all files"
+
+# If we are in enforcing, we should try to relabel before the reboot, because if some files
+# are not correctly labelled, system may not be able to start the autorelabel service (ie: /etc/localtime)
+
+test $( grep "SELINUX=enforcing" $TARGET_FS_ROOT/etc/selinux/config ) || return 0
+
+local policy=$( grep "^SELINUXTYPE" $TARGET_FS_ROOT/etc/selinux/config | sed 's/SELINUXTYPE=//' )
+
+LogPrint "We try to restore the selinux labels before the first reboot because the configuration is enforcing and autorelabel may fail. \n
+        This can take a several minutes."
```

Comment:
For me this kind of LogPrint argument does not result what seems to be intended
because I added to a script (init/default/030_update_recovery_system.sh)
```
LogPrint "line1\n
    line2"
LogPrint "line1
line2"
```
and I got
```
# usr/sbin/rear -D mkrescue
...
Running 'init' stage ======================
line1\n
    line2
line1
line2
```